### PR TITLE
Feature: extend L.esri.Cluster.featureLayer too

### PR DESCRIPTION
**Pull request by jgravois**
resolves https://github.com/Esri/esri-leaflet-cluster/issues/18

i had a little time today so i came up with a rough cut of how this plugin could be extended to ensure that `L.esri.Cluster.featureLayer` will draw the symbology defined in a service any time a feature _isn't_ clustered too.

the check for `L.esri.Cluster` feels like a hack and i don't think it would work with Webpack, but using ES6 to import `Cluster` as a dependency doesn't seem like the right thing either.

### CODE REVIEW

File: src/ClusterFeatureLayerHook.js [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+import simpleRenderer from './Renderers/SimpleRenderer';
+
+if (L.esri.Cluster) {
+  L.esri.Cluster.FeatureLayer.addInitHook(function () {
+    if (this.options.ignoreRenderer) {
+      return;
+    }
+    var oldOnAdd = L.Util.bind(this.onAdd, this);
+    var oldUnbindPopup = L.Util.bind(this.unbindPopup, this);
+    var oldOnRemove = L.Util.bind(this.onRemove, this);
```
Comment (jgravois) [inline src/ClusterFeatureLayerHook.js]: literally the only difference between this new hook and the existing one is to _not_ bind `this.createNewLayer`.

**Comment by jgravois** [comment]
yolo

File: package.json [hunk 2]
```diff
@@ -11,38 +11,45 @@
     "John Gravois <jgravois@esri.com>"
   ],
   "dependencies": {
-    "esri-leaflet": "^2.0.0",
-    "leaflet": "^1.0.0-rc.3",
-    "leaflet-shape-markers": "^1.0.4"
+    "esri-leaflet": "^2.2.3",
+    "leaflet": "^1.3.4",
+    "leaflet-shape-markers": "^1.0.6"
```
Comment (jgravois) [inline package.json:16]: as discussed in https://github.com/Esri/esri-leaflet-renderers/issues/152, _loose_ peerDependencies would be a better idea.

**Comment by gavinr** [comment]
Now available in https://github.com/Esri/esri-leaflet-renderers/releases/tag/v2.1.1

**Pull request merged**